mod_reforged/hooks/skills/skill.nut

```diff
@@ -26,7 +26,6 @@
 				// We remove these entries from ever displaying because they are mostly redundant and take away space
 				case "Fast Adaption":		// This should be handled by the perk itself if it really feels like this information is important
 				case "Armed with shield":	// Already displayed on the stats of the character and shield is visible
-				case "Shieldwall":			// Already displayed on the stats of the character and in the effects section
```

Comment:
Those stats and effects display are gated behind mod settings, so someone can potentially turn that off. I think this should not be removed.

Reply:
The line in question removes the vanilla entry with `"Shieldwall"`
Vanilla no longer adds any entry called `"Shieldwall"`
Therefor that line is now redundant

Reply:
![image](https://github.com/user-attachments/assets/a9bbb713-0b88-4892-b439-9cfe88589f10)

Reply:
I see, I misunderstood it then. Thanks for the clarification, looks good.